src/common/rdtsc.cpp

```diff
@@ -55,6 +58,12 @@ u64 EstimateRDTSCFrequency() {
     const u64 tsc_diff = tsc_end - tsc_start;
     const u64 tsc_freq = MultiplyAndDivide64(tsc_diff, 1000000000ULL, end_time - start_time);
     return RoundToNearest<100'000>(tsc_freq);
+
+    #else
+
+    return 0;
```

Comment:
Should have an alternate implementation or at least a TODO note.

Reply:
Done